manifests/init.pp

```diff
@@ -1,4 +1,4 @@
-# @summary Base Sensu class
+# n@summary Base Sensu class
```

Comment:
This looks like a typo

Reply:
Yep, vim and multi tasking didn't mix. I fixed this.